### PR TITLE
Improve appearance

### PR DESCRIPTION
**Pull request by jorgectf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+6. Commit and push your changes
+7. Open a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+
+### Suggesting changes
+
+Even though this place is built from community pull requests, if you don't want to go through the hassle of forking and cloning, you can also suggest changes by opening an [issue](https://github.com/jorgectf/ctf-web/issues/new?assignees=&labels=&projects=&template=suggestion.yml&title=%5BSuggestion%5D%20).
+
+## Contributors
+
+- [maikypedia]()
```
Comment (jorgectf) [inline README.md]: @maikypedia please add a link for your handle

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+2. Clone your fork
+3. Make your changes
+4. Add yourself to the [contributors](#contributors) list
+5. Commit and push your changes
+6. Open a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+
+### Suggesting changes
+
+Even though this place is built from community pull requests, if you don't want to go through the hassle of forking and opening a pull request, you can also suggest changes by opening an [issue](https://github.com/jorgectf/ctf-web/issues/new?assignees=&labels=&projects=&template=suggestion.yml&title=%5BSuggestion%5D%20).
+
```
Comment (maikypedia) [inline README.md]:
```suggestion
```

I think it will be more understandable if we put this section in CONTRIBUTING.md.

**Comment by maikypedia** [comment]
@jorgectf If we want it to be stable in the long term I think it would be a good idea adding this to `CONTRIBUTING.md`:

### Adding writeups

We do not host any writeups directly on the gitbook. Instead, we provide links to the writeup author's blog. Additionally, for added stability and access to historical versions, we use [Web Archive](http://web.archive.org/) links as an alternative means to access content. With this approach, we ensure that if there are any changes to the author's blog, like domain name or URL alterations in the future, you'll still be able to access the content through the provided links.

**Review by maikypedia — CHANGES_REQUESTED** [review]

**Review by maikypedia — APPROVED** [review]
LGTM

**Pull request merged**